README.md

```diff
@@ -1,9 +1,31 @@
+
 # Marketing Pages
 
+  
+
 Nodejs with Express app to present the marketing pages.
 
+  
+
 ## Prerequisite
 
-Install the long-term support (LTS) version of <a href="https://nodejs.org/en/">Node.js</a>, which includes NPM.
+  
+
+Install the long-term support (LTS) version of <a  href="https://nodejs.org/en/">Node.js</a>, which includes NPM.
+
+  
 
 ## Running the application
+
+ 1. Make sure you have the API running
+ 2. export env. variable `export NODE_ENV='development'`
```

Comment:
I believe this is a bash command to set an environment variable. Please consider other developers will be using a different shell e.g. Powershell.

See the readme on the MarketingPagesAcceptanceTests repository for an example of other formats...

https://github.com/nhs-digital-gp-it-futures/MarketingPageAcceptanceTests